packages/firebase_data_connect/firebase_data_connect/lib/src/network/grpc_transport.dart

```diff
@@ -59,9 +63,11 @@ class GRPCTransport implements DataConnectTransport {
     } catch (e) {
       log('Unable to get app check token: $e');
     }
+    String sdkTypeValue =
+        "flutter/${sdkType == ClientSDKType.core ? 'core' : 'gen'}";
```

Comment:
I think the value here would be flutter/core and flutter/gen. 

This blank if sdk is core. 

If gen the format is lang/gen => dart/gen

Reply:
Pinged you the internal chat thread on this